internal/command/command.go

```diff
@@ -340,6 +346,25 @@ func handleError(description string, err error) {
 	os.Exit(1)
 }
 
+// checkVersion fetches latest version and compares it to local
+func checkVersion(logger output.Logger) {
+	resp, err := http.Get("https://raw.githubusercontent.com/onflow/flow-cli/master/version.txt")
+	if err != nil || resp.StatusCode >= 400 {
+		return
+	}
+
+	defer resp.Body.Close()
+	latestVersion, _ := ioutil.ReadAll(resp.Body)
+
+	if string(latestVersion) != build.Semver() {
+		logger.Info(fmt.Sprintf(
+			"\n⚠️  Version Warning: New version %s of Flow CLI is available.\n"+
+				"Follow the Flow CLI installation guide for instructions on how to install or upgrade the CLI: https://docs.onflow.org/flow-cli/install",
+			strings.ReplaceAll(string(latestVersion), "\n", ""),
```

Comment:
I would sanitize this string before the `if` statement, just in case. You can also use `strings. TrimSpace` for this, I think.

```go
body, _ := ioutil.ReadAll(resp.Body)

latestVersion := strings.TrimSpace(string(body))

if latestVersion != build.Semver() {